client/src/document/index.scss

```diff
@@ -93,11 +93,10 @@ a.new {
     }
   }
 
-  a.self-link {
+  a[aria-current="true"] {
+    color: $neutral-100;
     font-weight: bold;
-    // font-style: italic;
     text-decoration: none;
-    color: black;
```

Comment:
I reckon we can make this:

```css
[aria-current] {
    color: $neutral-100;
    font-weight: bold;
    text-decoration: none;
  }
```

Then if we want to treat the `page` and `true` states differently in the future, we can update this.